docs/en/latest/discovery/zookeeper.md

```diff
@@ -0,0 +1,114 @@
+---
+title: nacos
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Service discovery via Zookeeper
+
+`Zookeeper` service discovery needs to rely on the [apisix-seed](https://github.com/api7/apisix-seed) project.
+
+### How apisix-seed works
+
+![APISIX-SEED](../../../assets/images/apisix-seed.png)
+
+### Setting apisix-seed and zookeeper
+
+The configuration steps are as follows:
+
+1. Download and compile the `apisix-seed` project.
+
+```bash
+$ git clone https://github.com/api7/apisix-seed.git
+$ go build
+```
+
+2. Start the Zookeeper service
+
+```bash
+$ docker run -itd --rm --name=dev-zookeeper -p 2181:2181 zookeeper:3.7.0
+```
+
+4. Modify the `apisix-seed` configuration file.
+
+```bash
+$ vim conf/conf.yaml
+etcd:                            # APISIX ETCD Configure
+  host:
+    - "http://127.0.0.1:2379"
+  prefix: /apisix
+  timeout: 30
+
+discovery:
+  zookeeper:                     # Zookeeper Service Discovery
+    hosts:
+      - "127.0.0.1:2181"
+    prefix: /zookeeper
+    weight: 100                  # default weight for node
+    timeout: 10                  # default 10s
+```
```

Comment:
I don't think it's a good idea to put the command and return in the same block of code.